gulp/tasks/scripts.js

```diff
@@ -21,23 +24,32 @@ const paths = require( '../../config/environment' ).paths;
 const webpack = require( 'webpack' );
 const webpackConfig = require( '../../config/webpack-config.js' );
 const webpackStream = require( 'webpack-stream' );
-const configLegacy = require( '../config.js' ).legacy;
+
+const NEWER_EXTRAS = [
```

Comment:
Can we change the name to something like `ADDITIONAL_SCRIPTS` or `ADDITIONAL_FILES`?

Reply:
Also, we should probably just move this to the config file.

Reply:
> Can we change the name to something like `ADDITIONAL_SCRIPTS` or `ADDITIONAL_FILES`? I like the connection to the property name in the gulp-newer options, but it could be clearer. How about `EXTRA_FILES_FOR_GULP_NEWER`?

Reply:
No, because if you don't use `gulp_newer` then the variable name no longer makes sense. Especially if you move the variable into a config file.

Reply:
> Also, we should probably just move this to the config file.

I disagree with this. This variable will only ever be used within this task file. Moving it to config just adds that much more overhead when debugging or just trying to understand this setup in the future.

Reply:
> No, because if you don't use `gulp-newer` then the variable name no longer makes sense. Especially if you move the variable into a config file.

Good point, but `ADDITIONAL_FILES` isn't clear enough. Additional files for what?

Reply:
>> I disagree with this. This variable will only ever be used within this task file. Moving it to config just adds that much more overhead when debugging or just trying to understand this setup in the future.

right, but this would be the only config object in the gulp files and the code isn't reusable. We should stay with the existing convention and move them to the config file.

Reply:
OK, moved these things to config and renamed them to be tool-agnostic in a3f4425.